### ISSUE TITLE
File Upload example bug

### ISSUE THREAD

**Issue by akelyasir** [teodevgroup/teo#62]
@victorteokw 

While reviewing the sample codes, I got an error again.

```
@map(path: "/upload")
declare form handler upload(UploadInput): Data<UploadOutput>
 
interface UploadInput {
  file: File
}
 
interface UploadOutput {
  path: String
}
```

It gives an error after the entity is created.

```
error[E0599]: no function or associated item named `from_value_ref` found for struct `teo::prelude::File` in the current scope
     --> src/entities/mod.rs:16999:15
      |
16999 |         File::from_value_ref(self.inner().get("file").unwrap()).unwrap()
      |               ^^^^^^^^^^^^^^ function or associated item not found in `File`

```

I also want to use a structure like this as a field in the schema file. But it also gives errors.

```
@relation(fields: .id, references: .id)
 contacts: User[]?
```

```
error[E0609]: no field `inner` on type `Vec<User>`
     --> src/entities/mod.rs:13500:77
      |
13500 |         self.inner.force_set_relation_object("contacts", contacts.map(|o| o.inner.clone())).await
      |                                                                             ^^^^^ unknown field
```

**Comment by victorteokw** [comment]
Hi @akelyasir, I will fix this bug. I'm currently moving my house. Sorry for this.

**Comment by akelyasir** [comment]
Hİ @victorteokw 

I hope your new home will be beautiful. I found a solution for this problem, but I don't know if it is the right solution.

I added the following code to this file **/src/entities/helpers/interface.rs.** Also, can we use the multipart feature for uploading? I couldn't find a solution for this. Finally, does it make sense to use unsafe?

`impl AsInterfaceRef for File {

    fn from_value_ref(value: &Value) -> Result<&File> {
        Ok(unsafe {
            &*(value as *const Value as *const File)
        })
    }
}
`

**Comment by victorteokw** [comment]
I will update the example

**Comment by victorteokw** [comment]
Hi @akelyasir, temporarily do this, I will update a patch version of the framework.
```rust
pub trait UploadInputTrait: Interface {
    /// ## File
    ///
    /// This interface field doesn't have a description.
    fn file(&self) -> &File {
        self.inner().get("file").unwrap().as_file().unwrap()
    }
    /// ## File
    ///
    /// This interface field doesn't have a description.
    fn set_file(&mut self, new_value: File) {
    self.inner_mut().as_dictionary_mut().unwrap().insert("file".to_owned(), new_value.into()).unwrap();
    }
}
```

**Comment by victorteokw** [comment]
This bug is fixed in Teo 0.2.36
